### DISCUSSION TITLE
configure diagnostics level

### DISCUSSION THREAD

**Discussion by lone1y-51** [LazyVim/LazyVim#2034] [Q&A] [answered]
how to disable warning level's diagnostics?
![image](https://github.com/LazyVim/LazyVim/assets/11805860/b5303ee1-0b7c-4b82-88f3-8be17313b5c1)

**Comment by dpetka2001** [comment, marked as answer]
I haven't found a way to exclude a severity level (though that might be due to my lack of knowledge, so would welcome a more complete solution). So, the next best thing would be to include only severity of ERROR level. You could do something like 
```lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      diagnostics = {
        virtual_text = {
          severity = vim.diagnostic.severity.ERROR,
        },
      },
    },
  },
}
```
in a file in `~/.config/nvim/lua/plugins/lsp.lua`.

**Reply by lone1y-51** [reply]
tks
